boards/google/icetower/board.yml

```diff
@@ -0,0 +1,6 @@
+board:
+  name: google_icetower
+  full_name: Google Icetower Development Board
```

Comment:
```suggestion
  full_name: Icetower Development Board
```

Reply:
see https://builds.zephyrproject.io/zephyr/pr/83034/docs/boards/index.html#vendor=google -- vendor name is redundant in the full name

Reply:
Done.